bin/generate_gene_counts.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import argparse
+import re
+from collections import OrderedDict
+from pathlib import Path
+
+import pandas as pd
+
+translator = {
+    "unstranded": 1,
+    "forward": 2,
+    "reverse": 3,
+}
+
+
+def get_non_std_genes(gtf: str) -> set[str]:
+    """Create list of genes not belonging to chr1-21 or chrM"""
```

Comment:
If I remember correctly from Vincente this has been fixed as of the later versions of drop. When I ran with bam files I didn't filter for odd scaffolds, could be different for count matrices though.

Reply:
Right now this is also hard coded to hg38. Which should be fine ;)